integrations/adk-py/examples/hello_eval.py

```diff
@@ -0,0 +1,22 @@
+import asyncio
+from typing import Any
+
+from braintrust.framework import EvalAsync
+from braintrust_adk import setup_braintrust
+from hello import main as hello
+
+
+async def main():
+    async def task(input: Any):
+        setup_braintrust()
```

Comment:
is this supposed to happen in the task or did you mean to put it at the top level?

Reply:
it can go anywhere and multiple calls are noop. other parts explain top level, so i think we could leave it as is